meta-integrity/recipes-kernel/linux/linux/0001-ima-fix-ima_inode_post_setattr.patch

```diff
@@ -0,0 +1,35 @@
+From 6cfd84f7e581b8cade257032d97d29e5e502a47b Mon Sep 17 00:00:00 2001
+From: Mimi Zohar <zohar@linux.vnet.ibm.com>
+Date: Mon, 29 Feb 2016 22:00:13 -0500
+Subject: [PATCH] ima: fix ima_inode_post_setattr
+
+Changing file metadata (eg. uid, guid) could result in having to
+re-appraise a file's integrity, but does not change the "new file"
+status nor the security.ima xattr.  The IMA_PERMIT_DIRECTIO and
+IMA_DIGSIG_REQUIRED flags are policy rule specific.  This patch
+only resets these flags, not the IMA_NEW_FILE or IMA_DIGSIG flags.
+
+With this patch, changing the file timestamp will not remove the
+file signature on new files.
+
+Reported-by: Dmitry Rozhkov <dmitry.rozhkov@linux.intel.com>
+Signed-off-by: Mimi Zohar <zohar@linux.vnet.ibm.com>
+---
```

Comment:
Upststream-Status: Submitted [link to IMA mailing list post] needs to be added. If it's already in some kernel tree, even better.